src/Lowranksvd.jl

```diff
@@ -1,5 +1,163 @@
 module Lowranksvd
+# Low Rank Singular Value Decomposition
+using LinearAlgebra
+import Base.show
+"""
+    LowRankSVD <: Factorization
+Matrix factorization type of the singular value decomposition (SVD) approximation of a low rank matrix `A`.
+LowRankSVD is return type of [`svd_lowrank(_)`](@ref), the corresponding matrix factorization function.
+If `F::SVD` is the factorization object, `U`, `S`, `V` and `Vt` can be obtained
+via `F.U`, `F.S`, `F.V` and `F.Vt`, such that `A ≈ U * Diagonal(S) * Vt`.
+The singular values in `S` are sorted in descending order.
+"""
+struct LowRankSVD{T,Tr,M<:AbstractArray{T}} <: Factorization{T}
+    U::M
+    S::Vector{Tr}
+    Vt::M
+    function LowRankSVD{T,Tr,M}(U, S, Vt) where {T,Tr,M<:AbstractArray{T}}
+      size(U,2) == size(Vt,1) == length(S) || throw(DimensionMismatch("$(size(U)), $(length(S)), $(size(Vt)) not compatible"))
+      new{T,Tr,M}(U, S, Vt)
+    end
+end
+LowRankSVD(U::AbstractArray{T}, S::Vector{Tr}, Vt::AbstractArray{T}) where {T,Tr} = LowRankSVD{T,Tr,typeof(U)}(U, S, Vt)
+function LowRankSVD{T}(U::AbstractArray, S::AbstractVector{Tr}, Vt::AbstractArray) where {T,Tr}
+    LowRankSVD(convert(AbstractArray{T}, U),
+                convert(Vector{Tr}, S),
+                convert(AbstractArray{T}, Vt))
+end
+function Base.show(io::IO, mime::MIME{Symbol("text/plain")}, F::LowRankSVD{<:Any,<:Any,<:AbstractArray})
+    summary(io, F); println(io)
+    println(io, "U factor:")
+    show(io, mime, F.U)
+    println(io, "\nsingular values:")
+    show(io, mime, F.S)
+    println(io, "\nVt factor:")
+    show(io, mime, F.Vt)
+end
+"""
+    get_approximate_basis(A, l::Int64; niter::Int64 = 2, M = nothing) -> Q
+
+Return Matrix ``Q``` with ``l`` orthonormal columns such that ``Q Q^H A`` approximates ``A``. If ``M`` is specified, then ``Q`` is such that ``Q Q^H (A - M)`` approximates ``A - M``.
```

Comment:
应该单个 ` 就足够了？（还有一些其他的地方不过我没有标出来）

```suggestion
Return Matrix `Q` with `I` orthonormal columns such that `Q Q^H A` approximates `A`. If `M` is specified, then `Q` is such that `Q Q^H (A - M)` approximates `A - M`.
```

Reply:
两个`意思是数学表达式，这样， \``Q Q^H A`` 才能被正确的在文档中正确显示。

Reply:
抱歉（手动捂脸）是的，``用于数学表达式，`用于代码块，但我有些部分手滑了多打了一个引号...